way_to_home/static/src/components/header/header.js

```diff
@@ -1,18 +1,59 @@
 import React, {Component} from 'react';
+import {withRouter} from 'react-router-dom';
+import { withStyles } from '@material-ui/core/styles';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import AccountCircle from '@material-ui/icons/AccountCircle';
+import Avatar from '@material-ui/core/Avatar';
+import ToggleDisplay from 'react-toggle-display';
+
 import LoginBtn from '../loginBtn/loginBtn.js';
+import StartBtn from '../startBtn/startBtn.js';
+import SettingsButton from './settingsButton/settingsButton.js';
 import './header.css'
-import StartBtn from "../startBtn/startBtn.js";
 
+const style = {
+    height: '44px',
+    background: '#4887c1cc',
+    borderRadius: 5,
+    display: 'flex',
+};
```

Comment:
```suggestion
const style = {
    height: '44px',
    background: '#4887c1cc',
    borderRadius: 5,
    display: 'flex',
};
```
Are you sure that u need this styles in this file?

Reply:
I can't find another way to redefine style of 'AppBar' class. With the usual method such as using css it not work.